tests/test_grains_minicircle.py

```diff
@@ -46,13 +46,13 @@ def test_mask_minicircle(minicircle_grain_mask: Grains, plotting_config: dict, p
     return fig
 
 
-@pytest.mark.mpl_image_compare(baseline_dir="resources/img/")
+#@pytest.mark.mpl_image_compare(baseline_dir="resources/img/")
```

Comment:
Sorry, I missed this when I looked at it previously but why are the fixtures commented out here and in other tests that generate images?

Reply:
Oops, I think it was back when I was looking at specific test outcomes and I forgot to change them back. Ammended now.